lib/services/sqlManagement2/package.json

```diff
@@ -7,7 +7,10 @@
     "ms-rest": "^2.3.3",
```

Comment:
This requires at least a major version bump.

Reply:
We are moving from '2015-05-01-preview' to '2017-03-01-preview', so yeah let's do that.